### PR TITLE
add documentation for beta stability levels

### PR DESCRIPTION
**Pull request by logicalhan**
As per [KEP-3498](https://github.com/kubernetes/enhancements/issues/3498)

### CODE REVIEW

File: content/en/docs/reference/using-api/deprecation-policy.md
Comment (sftim) [inline content/en/docs/reference/using-api/deprecation-policy.md:1]: :warning: to reviewers

This change is adding a new “internal” phase into the deprecation policy.
We should confirm this with SIG Architecture.

**Comment by sftim** [comment]
I recommend splitting out autogenerated updates and other changes (use separate commits).

File: content/en/docs/reference/using-api/deprecation-policy.md
Comment (logicalhan) [reply content/en/docs/reference/using-api/deprecation-policy.md:1]: It was actually added by some sig arch members suggestions. But currently we have no internal metrics so I can actually just omit this for now.

**Comment by logicalhan** [comment]
> I recommend splitting out autogenerated updates and other changes (use separate commits).

Done.

**Pull request merged**